packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -367,6 +374,25 @@ export class CrossChainMessenger {
     }
   }
 
+  public async getWithdrawalsFromMessage(message: MessageLike) {
```

Comment:
I normally would make this private but our sdk has many other similar methods to this public so staying consistent